03week/airplane/index.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Airplane</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <!-- For some reason, specifying font in CSS broke my header, so I didn't -->
+  <link rel="stylesheet" href="./css/reset.css" />
+  <link rel="stylesheet" href="./css/style.css" />
+</head>
+
+<body>
+  <header>
+    <div class="NavBar">
```

Comment:
good use of classes.. if I can give a suggestion -- try to start classes and id's with lowercase letters.. then use camel case as you are or underscores. Capitalized things mean something different later on down the line so it's good to get in the habit of using lowercase. 

i.e. `class="navBar"` or `class="nav_bar"` (the first one is usually seen more)